phoenix-shared/src/commonMain/paymentsdb/fr.acinq.phoenix.db/AggregatedQueries.sq

```diff
@@ -78,4 +78,34 @@ LIMIT ?, ?;
 -- 14: incoming_received_with
 -- 15: incoming_received_with_fees
 -- 16: created_at                       payment creation timestamp
--- 17: completed_at                     payment completion timestamp
\ No newline at end of file
+-- 17: completed_at                     payment completion timestamp
+
+listAllPaymentsOrder:
+SELECT *
+FROM (
+    SELECT
+        'outgoing'              AS direction,
+        id                      AS outgoing_payment_id,
+        NULL                    AS incoming_payment_id,
+        created_at              AS created_at,
+        completed_at            AS completed_at
+    FROM outgoing_payments
+UNION ALL
+    SELECT
+        'incoming'              AS direction,
+        NULL                    AS outgoing_payment_id,
+        payment_hash            AS incoming_payment_id,
+        created_at              AS created_at,
+        received_at             AS completed_at
+    FROM incoming_payments
+    WHERE received_amount_msat > 0
+)
+ORDER BY COALESCE(completed_at, created_at) DESC
+LIMIT ?, ?;
```

Comment:
nit: use the `OFFSET` optional keyword, as recommended by the [sqlite docs](https://www.sqlite.org/lang_select.html#the_limit_clause).

As a side note, I think it would be better to use the seek method for pagination instead of an offset. But that may not be worth the effort considering it's unlikely the user would scroll back far enough in time to affect performances.

Reply:
Done in bc1998e